src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Item;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Item> items = new ArrayList<>();
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```